src/components/Cart/CartProducts/CartProducts.vue

```diff
@@ -16,7 +16,30 @@
     <ProductsHover v-else :item="item" />
   </div>
 
-  <div v-if="cart.length === 0" class="np-cartProducts__empty">
+  <!--  Pagination-->
+  <div class="np-cartProducts__pagination" v-if="page.length">
+    <Arrow
+      @click="prevPage"
+      class="pagination__arrow pagination__arrow--left"
+    />
+    <label for="pageNumber">
+      <input
+        :value="inputValue"
+        @change="setPage"
+        type="text"
+        id="pageNumber"
+        name="pageNumber"
+        class="pagination__input"
+      />
+    </label>
+    <Arrow
+      @click="nextPage"
+      class="pagination__arrow pagination__arrow--right"
+    />
+  </div>
+  <!--End of pagination-->
+
+  <div v-if="!page.length" class="np-cartProducts__empty">
```

Comment:
if youre using BEM convention then empty should be a modifier (--empty not __empty)